01-fundamentals/first_letter.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2016 Ami . All rights reserved.
+
+def get_user_password():
+    password = input("Tell me your password:")
+    return password
+
+
+def show_first_letter_of(password):
```

Comment:
`print_` will be more suitable here. Also, the `_of` postfix is not really required but it's a nice touch.